### ISSUE TITLE
Stop GW from minimising then maximising on map change when in full screen mode

### ISSUE THREAD

**Issue by 3vcloud** [gwdevhub/GWToolboxpp#1042]
When in full screen mode, GW seems to dissappear then reappear in the process of a frame, which is a bit disorienting. If its an easy fix like maybe blocking a WndProc or something, that would be nice

**Comment by AlexMacocian** [comment]
I don't think it minimizes. It seems to instead show some old video buffer. Try starting the game in borderless mode and then switch to fullscreen and start hopping maps.
You'll see that the game keeps showing an image from the moment you switched to full screen while loading the new scene

**Comment by AlexMacocian** [comment]
Ok so when you're in FullScreen, WM_WINDOWPOSCHANGED and WM_WINDOWPOSCHANGING messages are being triggered when switching maps. This doesn't happen in windowed mode. I'll look into how to block these

**Comment by AlexMacocian** [comment]
Ok played a bit with it but can't get it to affect anything. lparam is a WINDOWPOS*. This struct has a flags field. Inside, on the WM_WINDOWPOSCHANGING, the value is 0x34, equivalent to NOZORDER, NOACTIVATE, FRAMECHANGED.
In WM_WINDOWPOSCHANGED, the value is 0x1837, equivalent to NOSIZE, NOMOVE, NOZORDER, NOACTIVATE, FRAMECHANGED and two more flags that I couldn't find the value of.

- I tried resetting NOACTIVATE and FRAMECHANGED but it makes no difference...
- Tried fully blocking these messages when the WINDOWPOS size is the same as the client size, it doesn't solve the issue. It just breaks the window functionality (kinda expectedly) when minimizing and restoring
